applications/openshift/networking/ingress_controller_tls_security_profile_not_old/rule.yml

```diff
@@ -0,0 +1,62 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Ensure IngressController is not configured to use Old tlsSecurityProfile"
+
+description: |-
+    The configuration <tt>tlsSecurityProfile</tt> specifies TLS configurations
+    to be used while establishing connections with the externally exposed
+    servers. Though secure transport mode is used for establishing connections,
+    the protocols used may not always be strong enough to avoid interception and
+    manipulation of the data in transport. TLS Security profile Old should be
+    avoided, as it supports vulnerable protocols, ciphers, and algorithms which
+    could lead to security breaches.
+
+    To update tlsSecurityProfile from Old to Intermediate use the following command:
+    {{% raw %}}<pre>oc patch -n openshift-ingress-operator ingresscontrollers.operator.openshift.io default --type 'json' --patch '[{"op": "add", "path": "/spec/tlsSecurityProfile/intermediate", "value": {}}, {"op": "replace", "path": "/spec/tlsSecurityProfile/type", "value": "Intermediate"}, {"op": "remove", "path": "/spec/tlsSecurityProfile/old"}]'</pre>{{% endraw %}}
```

Comment:
hmm why do you use `raw` here? I only now noticed that you used that in the previous patch as well, why is `<pre>` not sufficient?

Reply:
Patch command was adding `"/spec/tlsSecurityProfile/intermediate": "value": {}` and to avoid `{}` formatting thought to use `raw`. I am not sure if `pre` alone will be sufficient.

Reply:
ok, thanks let me try how the resulting rule looks like in HTML guides and when rendered in the cluster

Reply:
Seems to be looking good!